bindings/python/convert.py

```diff
@@ -218,7 +218,7 @@ def convert_file(
     sf_filename: str,
     discard_names: List[str],
 ):
-    loaded = torch.load(pt_filename, map_location="cpu")
+    loaded = torch.load(pt_filename, map_location="cpu", weights_only=True)
```

Comment:
This will break pytorch lightning saved weights (which use a full dictionnary with "state_dict" for the actual state dict, and iirc (and unless they changed) they also save some custom lightning objects which are not weights only.